instagram/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import PostContainer from './PostContainer';
+import dummyData from './dummy-data'
+import SearchBar from './SearchBar'
+
+
+class App extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      posts:dummyData,
+    } 
+  }
+
+  inputHandler = event => {
+    const value = event.target.value;
+
+    this.setState (
+      {search:value}
+    );
+    }
+
+
+    submitHandler = event => {
+      event.preventDefault();
+  
+        const obj = {
```

Comment:
This obj may not be necessary. That is the use case of adding these key value pairs to state?